wiki/BBCode/en.md

```diff
@@ -243,7 +243,9 @@ The `[img]` tag is used to include online images into osu! forum posts. To use t
 
 In order to obtain an image address, one must navigate to the website it is sourced on, hover their mouse over the image, right-click on the image, and select `Copy image address`. Then, the address should be copied and pasted in between the tags.
 
-Although images can be sourced from anywhere, osu! recommends that users upload images to reputable image sharing sites like [Imgur](https://imgur.com), as some websites do not appreciate direct links to their images (otherwise known as "hotlinks").
+Although images can be sourced from anywhere, osu! recommends that users upload images to reputable image sharing sites like [ImgBB](https://imgbb.com/), as some websites do not appreciate direct links to their images (otherwise known as "hotlinks").
+
+*Notice: Imgur has blocked the IPs of the osu! website, so the images hosted there can no longer be displayed.*[^imgur-blocked-ip]
```

Comment:
they can be displayed since osu is mirroring all its instances of imgur images

Reply:
I guess it just mirrors the images that were already uploaded, because I tried to put one and it didn't work. So I've changed it to just mention that the new images don't display.